modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/FDS/APIApplicationsTable.tsx

```diff
@@ -0,0 +1,173 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {
+	FrontendDataSet,
+	IFrontendDataSetProps,
+} from '@liferay/frontend-data-set-web';
+import React, {useEffect, useState} from 'react';
+
+import {getItems} from '../utils/fetchUtil';
+import {itemPathRenderer, itemStatusRenderer} from '../utils/renderers';
+
+interface APIApplicationsTableProps {
+	apiURL: string;
+	portletId: string;
+	readOnly: boolean;
+}
+
+export default function APIApplicationsTable({
+	apiURL,
+	portletId,
+	readOnly,
+}: APIApplicationsTableProps) {
+	const [ApiApplications, setApiApplications] = useState<ItemData[]>([]);
```

Comment:
@cgmonte Please, rename them to "APIApplications", "setAPIApplications" to keep the same pattern.